contracts/governance_standard/GovernorContract.sol

```diff
@@ -6,6 +6,7 @@ import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol
 import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
```

Comment:
What's the purpose of this change?

Reply:
Previously, received error as 

```
CompilerError: solc returned the following errors:

DeclarationError: Identifier not found or not unique.
  --> contracts/governance_standard/GovernorContract.sol:23:9:
   |
23 |         ERC20Votes _token,
   |         ^^^^^^^^^^

      ^^^^^^^^^^
```

But I just tried to repro with new a fresh `brownie bake dao-mix`, this error stopped happening, will remove it from this PR.